src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -137,6 +137,7 @@ namespace HVACVariableRefrigerantFlow {
     extern Real64 CompOnFlowRatio;               // fan flow ratio when coil on
     extern Real64 CompOffFlowRatio;              // fan flow ratio when coil off
     extern Real64 FanSpeedRatio;                 // ratio of air flow ratio passed to fan object
+    extern Real64 CurrentEndTimeLast;            // end time of last time step
```

Comment:
The issue was this local var to function CalcVRFCondenser was not getting reset from previous unit tests. The question is the best way to move this var up to the module level so that it can be reset using clear_state. Does this var need to be in the hh or can this simply be a non-external variable?